features/fixtures/mazerunner/src/main/java/com/bugsnag/android/mazerunner/scenarios/MultiProcessHandledExceptionScenario.kt

```diff
@@ -20,17 +20,21 @@ internal class MultiProcessHandledExceptionScenario(
 
     override fun startScenario() {
         super.startScenario()
-        Bugsnag.notify(generateException())
-
         if (!isRunningFromBackgroundService()) {
-            val scenarioParams = BugsnagIntentParams(
-                javaClass.simpleName,
-                config.apiKey,
-                config.endpoints.notify,
-                config.endpoints.sessions,
-                eventMetadata
-            )
-            launchMultiProcessService(scenarioParams::encode)
+            launchMultiProcessService(
+                BugsnagIntentParams.fromConfig(
+                    config,
+                    javaClass.simpleName,
+                    eventMetadata
+                )
+            ) {
+                Bugsnag.setUser("123", "jane@example.com", "Jane")
```

Comment:
Making sure this is more explicit on which user belongs to which process:
```suggestion
                Bugsnag.setUser("2", "background@example.com", "MultiProcessHandledExceptionScenario")
```